src/Amazon.Common.DotNetCli.Tools/AbstractCLIWrapper.cs

```diff
@@ -4,7 +4,7 @@
 using System.IO;
 using System.Text;
 
-#if NETCORE
+#if NETCOREAPP3_1_OR_GREATER
```

Comment:
Do we need this? Why are we using preprocessor directives NETCOREAPP3_1_OR_GREATER at all when the csproj targetframeworks look like we never build anything below 3.1 anyway?

Reply:
We need them because the source code is also absorbed by the Visual Studio Toolkit which uses .NET Framework.